docs/faq.md

```diff
@@ -4,6 +4,134 @@ title: ❓ FAQ
 description: Frequently asked questions.
 ---
 
+If you have questions not answered here, please ask! Both filing an issue
+or asking on [Discord](https://discord.gg/9hKJcWGcaB)) work.
+
+### What is "code push"?
+
+Code push, also referred to as "over the air updates" (OTA) is a cloud service
+we are building to enable Flutter developers to deploy updates directly to
+devices anywhere they have shipped Flutter. We've currently shown demos using
+Android, but since it's built with Flutter/Dart (and Rust) it can eventually
+work anywhere Flutter works.
+
+The easiest way to see is to watch the [demo
+video](https://www.youtube.com/watch?v=mmKvs0_Zu14&ab_channel=Shorebird).
+
+"Code Push" is a reference to the name of a deploy feature used by the React
+Native community from [Microsoft](https://appcenter.ms) and
+[Expo](https://expo.dev), neither of which support Flutter.
+
+### What is the roadmap?
+
+We try to keep: https://docs.shorebird.dev/status up to date with the status
+of the project.
+
+Our project boards are also public an found at:
+https://github.com/orgs/shorebirdtech/projects
+
+### How does this relate to Firebase Remote Config or Launch Darkly?
+
+Code push allows adding new code / replacing code on the device. Firebase
+Remote Config and Launch Darkly are both configuration systems. They allow you
+to change the configuration of your app without having to ship a new version.
+They are not intended to replace code.
+
+### How big of a dependency footprint does this add?
+
+I haven't measured recently, but I expect the code push library to add less than
+one megabyte to Flutter apps. `flutter build apk --release` vs.
+`shorebird build apk --release` should give you a rough idea. We know of ways
+we can make this smaller when that becomes a priority. If size is a blocker
+for you, please let us know!
```

Comment:
Filed https://github.com/shorebirdtech/docs/issues/19 so we can at least give a ballpark idea of the size difference, which should help readers determine whether/how much of a blocker this might be.